packages/html/src/globals.ts

```diff
@@ -0,0 +1,31 @@
+import { bigtestGlobals } from "@bigtest/globals";
+
+interface Globals {
+  documentResolver: () => Document;
+}
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace, @typescript-eslint/prefer-namespace-keyword
+  module globalThis {
+    // eslint-disable-next-line prefer-let/prefer-let, no-var
+    var __interactors: Globals;
+  }
+}
+
+if (!globalThis.__interactors) {
+  globalThis.__interactors = {
+    documentResolver: () => bigtestGlobals.document,
```

Comment:
Let's add a TODO for this PR to create a PR on bigtest to use `setDocumentResolver` once this is merged so that we can completely break the dependency on `bigtestGlobals.document`